src/components/calcite-dropdown/calcite-dropdown.scss

```diff
@@ -1,13 +1,11 @@
 // theme variables
```

Comment:
There are some updates to these styles on a pending PR, let's just make sure they get combined.

Reply:
sounds good